ament_copyright/ament_copyright/parser.py

```diff
@@ -248,6 +249,8 @@ def get_comment_block(content, index):
     comment_token = match.group(1)
     start_index = match.start(1)
 
+    if comment_token in ['/*', '/**']:
+        comment_token = ' *'
```

Comment:
I'm thinking something like this:

```py
if comment_token in ['/*', '/**']:
    comment_token = '*'
"""
more code
"""
lines = [line[line.index(comment_token) + 1:] for line in lines]
```

That works for the existing copyright cases, but not for the new ones.